index.js

```diff
@@ -1,25 +1,179 @@
-const navBar = document.querySelector('#nav-bar');
-const menuBtn = document.querySelector('#menu-btn');
-const exitBtn = document.querySelector('#exit-btn');
-const links = document.querySelectorAll('.nav-item');
-    
 
+const popUpCard = document.querySelectorAll('[data-modal-target]');
+const cardElement = document.querySelector('.card');
+
+const cardArray =  [
+  {
+    id: 'card-1',
+    title: 'Tonic',
+    list: {
+      title: 'canopy',
+      dev: 'Backend-dev',
+      year: '2015'
+    },
+    images: {
+      large: './images/Portfolio3x.png',
+      small: './images/Portfolio.jpg'
+    },
+    information: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    technologies: {
+      html: 'html',
+      css: 'css',
+      javascript: 'javascript',
+      ruby: 'ruby on rails',
+      github: 'github',
+      bootstrap: 'bootstrap'
+    },
+    live: 'https://munyabelden.github.io/',
+    source: 'https://github.com/Munyabelden/munyabelden.github.io'
+  },
+  {
+    id: 'card-2',
+    title: 'Multi-post Stories',
+    list: {
+      title: 'canopy',
+      dev: 'Backend-dev',
+      year: '2015'
+    },
+    images: {
+      large: './images/Portfolio3x.png',
+      small: './images/Portfolio.jpg'
+    },
+    information: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    technologies: {
+      html: 'html',
+      css: 'css',
+      javascript: 'javascript',
+      ruby: 'ruby on rails',
+      github: 'github',
+      bootstrap: 'bootstrap'
+    },
+    live: 'https://munyabelden.github.io/',
+    source: 'https://github.com/Munyabelden/munyabelden.github.io'
+  },
+  {
+    id: 'card-1',
+    title: 'Tonic',
+    list: {
+      title: 'canopy',
+      dev: 'Backend-dev',
```

Comment:
Good job trying to implement this but as suggested by the previous code reviewer your cards should be displayed dynamically using javascript DOM manipulation so all your code in your card section of your index.html file should be deleted and the cards should be rendered dynamically.

I also noticed your pop-up is only working on one card in your desktop version and not working in the last 3 cards and also in your mobile view your card are not popping up please kindly fix this issue for you to proceed to the next project both pop up windows for desktop and mobile view shouls be fully functional